### ISSUE TITLE
Accessibility of Maps

### ISSUE THREAD

**Issue by dauwhe** [daisy/kb#7]
We publish lots of different types of books that contain maps, from travel guides to fantasy novels (with maps of imaginary lands). 

Is there any specific advice for maps? Examples of best practices?

**Comment by mattgarrish** [comment]
Do you mean in terms of producing image maps, or how to describe images of maps?

**Comment by dauwhe** [comment]
Geographic maps, street maps, that kind of thing—set as images.

**Comment by mattgarrish** [comment]
Right, we don't have specific guidelines for how to author accessible descriptions. The first thing you generally have to ask yourself is what is the point of the image - what information does the user need to get from it.

If it's directions, for example, then a useful description would provide the steps to get to the destination. Or if the map is secondary to other information being imparted (e.g., if only certain cities or places are being highlighted), then you could list that information without the spatial context.

If it's to give a sense of a region, though, then the different areas could be described in relation to each other, but this often isn't very helpful. In these cases, tactile graphics are often a better option (provided the map isn't overly detailed) as they provide users who a better sense of dimension and proportion through touch.
